### ISSUE TITLE
Adding zoom buttons to leaflet map 

### ISSUE THREAD

**Issue by jarkenau** [naturerobots/HSOS-SEP-PlantMap-2022#126]
What's about adding zoom buttons like :heavy_plus_sign: and :heavy_minus_sign: to the leaflet map for people who don't use a mouse and whose trackpad is :wastebasket:? @lucaslootalot @dobro929 @fmarahre

**Comment by dobro929** [comment]
Yes, we have also thought about it. We will add the zoom buttons as soon as possible :thumbsup:
